portafly/src/i18n/locales/en/account_settings/personal/personal_details.yml

```
@@ -1,5 +1,74 @@
 ---
+# Account settings > Personal > Personal details
+
 page_title: Personal details | Portafly
-page_title_description: Page title of the subsection for Personal details
+page_title_description: Page title of the subsection for Personal details.
+
+# Top area
+
 body_title: Personal details
-body_title_description: Body title of the page for Personal details
+body_title_description: Body title of the page for Personal details.
+
+# Main area
+
+# Tabs
+
+user_data_title: User data
+user_data_description: This is the title of the form containing the user's personal information.
+change_password_title: Change password
+change_password_description: This is the title of the form containing the user's current password field and new password field.
+
+# User data form
+
+username_label: Username
+username_label_description: Label for the field that holds the current username.
+username_value: {{current_username}}
+username_value_description: This is the value contained in the username field.
+username_validation: {{username_validation}} is too short. Use a minimum of 3 characters. Do not use spaces. You can include letters, numbers, and the following special characters . - _ @ +
+username_validation_description: Input field that holds the current username, and validates the content.
+
+email_label: Email
+email_label_description: Label for the field that holds the valid user's email address.
+email_value: {{current_user_email}}
+email_value_description: By default, this is the email of the current user.
+email_validation: Use a valid email such as 'admin@domain.com'. The email can contain letters (both lower and upper case), numbers, and the following characters . - _ @ +
+email_validation_description: Input field that holds the valid user's email address.
+
+first_name_label: First name
+first_name_label_description: Label for the field that holds the user's first name.
+first_name_value: [User first name]
```

```suggestion
first_name_value: {{User_first_name}}
```

For default values, use this syntax.